### PR TITLE
Some tests

### PR DESCRIPTION
**Pull request by springcoil**
So thanks to @a96lex for the inspiration here. https://gist.github.com/a96lex/00ec7d0f4f9aa82cd23c89b5fc4d104f

I created with pytest a few test examples. This won't work until some open PR are merged note `Voice` not `Voice()`  (works now)

Some open questions
* How do we run this - do we use CircleCI/CD? 
* What other tests should we include that would also do this. I don't think we need automatic updating to pypi just yet. 
* Do we need some security checks with semgrep here in the circleci/cd. 
* How do we run these automatically? Or semi-automatically?

### CODE REVIEW

File: tests/test_config.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+import aflr
+from aflr import Script, Sound, Speech, Mastering, Voice
+import pprint
+import pytest
+
+aflr.api_key = ""
```
Comment (springcoil) [inline tests/test_config.py:7]: We need some environment variables here. @tonythree thoughts on how we should do this?

**Comment by zeritte** [comment]
> @zeritte Do you know ` # Still not sure how to get this to work properly assert city_nights[1].keys() == dict_keys(['citynights__music.wav'])` a good way to test dicts using pytest?

well city_nights can be None. so I would first check if it exists, then if its a list. then the length of it. and finally `assert list(city_nights[1]) == ['citynights__music.wav']` I would add. (https://stackoverflow.com/a/31837256)

**Comment by springcoil** [comment]
@zeritte Thanks I thought it was something like that.

**Review by FeaR6077 — APPROVED** [review]
lgtm

**Pull request merged**